templates/aws/index.html

```diff
@@ -1,160 +1,285 @@
+          </p>
+          <p>
+            Canonical continuously tracks and delivers updates to Ubuntu images to ensure security and stability are built-in from the moment your machines and containers launch.
+          </p>
           <p>Today, Ubuntu powers billions of mission-critical workloads.</p>
-          <a href="/aws/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on AWS</a>
         </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
-            alt="",
-            width="200",
-            height="119",
-            hi_def=True,
-            loading="auto",
-            ) | safe
+        <hr class="is-muted" />
+        <a href="/aws/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on AWS</a>
+      </div>
+      <div class="col">
+        <div class="p-image-container--16-9 is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
+                    alt="",
+                    width="123",
+                    height="74",
+                    hi_def=True,
+                    loading="auto",
+                    attrs={"class": "p-image-container__image"}) | safe
           }}
         </div>
       </div>
     </div>
   </section>
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
-      <h2 class="p-muted-heading">Ubuntu on AWS powers cloud leaders like</h2>
-      <div class="p-logo-section">
-        <div class="p-logo-section__items u-no-margin--bottom">
-          <div class="p-logo-section__item">
-            {{ image (
-              url="https://assets.ubuntu.com/v1/a6d62ef1-paypal-logo.png",
-              alt="PayPal",
-              width="288",
-              height="288",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-logo-section__logo"}
-              ) | safe
-            }}
-          </div>
-          <div class="p-logo-section__item">
-            {{ image (
-              url="https://assets.ubuntu.com/v1/584eb90e-heroku-logo.png",
-              alt="Heroku",
-              width="288",
-              height="288",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-logo-section__logo"}
-              ) | safe
-            }}
-          </div>
-          <div class="p-logo-section__item">
-            {{ image (
-              url="https://assets.ubuntu.com/v1/5f2fc96d-acquia-logo.png",
-              alt="Aquia",
-              width="288",
-              height="288",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-logo-section__logo"}
-              ) | safe
-            }}
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom" />
```

Comment:
makes sense to use a p-rule here

Reply:
`u-no-margin--bottom` is the replacement for `p-rule`, pr [here](https://github.com/canonical/vanilla-framework/pull/5241/files)